.travis.yml

```diff
@@ -6,12 +9,7 @@ rvm:
 env: DATABASE_URL=postgres://postgres@localhost/timeoverflow_test
 
 before_script:
-  - psql -c 'create database timeoverflow_test;' -U postgres
-  - RAILS_ENV=test bundle exec rake db:migrate
-
-sudo: false
-
-cache: bundler
+  - bundle exec rake db:setup db:migrate
```

Comment:
Every time we add a new migration, in the same PR we add the modified schema. We must ensure this. We generally enforce this running a `bad practices` suite. I would rely on our capability to don't break the schema, even more.. if we break the schema is better to know that in travis testing a PR than directly in production.

What do you think?